sdk/dotnet/Network/NetworkConnectionMonitor.cs

```diff
@@ -9,121 +9,53 @@
-    ///                 Port = 443,
-    ///             },
-    ///             Tags = 
-    ///             {
-    ///                 { "foo", "bar" },
-    ///             },
-    ///         }, new CustomResourceOptions
-    ///         {
-    ///             DependsOn = 
-    ///             {
-    ///                 srcExtension,
-    ///             },
-    ///         });
-    ///     }
-    /// 
-    /// }
-    /// ```
-    /// </summary>
     public partial class NetworkConnectionMonitor : Pulumi.CustomResource
     {
-        /// <summary>
-        /// Will the connection monitor start automatically once created? Changing this forces a new Network Connection Monitor to be created.
-        /// </summary>
         [Output("autoStart")]
-        public Output<bool?> AutoStart { get; private set; } = null!;
+        public Output<bool> AutoStart { get; private set; } = null!;
 
-        /// <summary>
-        /// A `destination` block as defined below.
-        /// </summary>
         [Output("destination")]
         public Output<Outputs.NetworkConnectionMonitorDestination> Destination { get; private set; } = null!;
 
         /// <summary>
-        /// Monitoring interval in seconds.
+        /// A `endpoint` block as defined below.
         /// </summary>
+        [Output("endpoints")]
+        public Output<ImmutableArray<Outputs.NetworkConnectionMonitorEndpoint>> Endpoints { get; private set; } = null!;
+
         [Output("intervalInSeconds")]
-        public Output<int?> IntervalInSeconds { get; private set; } = null!;
+        public Output<int> IntervalInSeconds { get; private set; } = null!;
 
         /// <summary>
-        /// The Azure Region where the Network Connection Monitor should exist. Changing this forces a new Network Connection Monitor to be created.
+        /// The Azure Region where the Network Connection Monitor should exist. Changing this forces a new resource to be created.
         /// </summary>
         [Output("location")]
         public Output<string> Location { get; private set; } = null!;
 
         /// <summary>
-        /// The name which should be used for this Network Connection Monitor. Changing this forces a new Network Connection Monitor to be created.
+        /// The name which should be used for this Network Connection Monitor. Changing this forces a new resource to be created.
         /// </summary>
         [Output("name")]
         public Output<string> Name { get; private set; } = null!;
 
         /// <summary>
-        /// The name of the Network Watcher. Changing this forces a new Network Connection Monitor to be created.
+        /// The ID of the Network Watcher. Changing this forces a new resource to be created.
         /// </summary>
-        [Output("networkWatcherName")]
-        public Output<string> NetworkWatcherName { get; private set; } = null!;
+        [Output("networkWatcherId")]
+        public Output<string> NetworkWatcherId { get; private set; } = null!;
```

Comment:
This bunch of changes is breaking too

Reply:
This is a flat out breaking change in their SDK... 

https://github.com/terraform-providers/terraform-provider-azurerm/pull/8640/files#diff-746c9ec7da44dc1d034172a67bfb5a97081472100c3c1eeb89f0e26e8fa704acR50